### PR TITLE
Extended search

### PR DESCRIPTION
**Pull request by litvinovg**
[Vitro PR](https://github.com/vivo-project/Vitro/pull/382)

# What does this pull request do?
Added extended search to VIVO

# What's new?
Added some document modifiers to index data in Solr fields
Changed Freemarker templates to use extended search by default.
Removed bootstrap dependency from tenderfoot theme - migrated to Vitro repository.

# How should this be tested?
Add dateRange field type to your Solr schema/managed-schema:
`  <dynamicField name="*_drsim" type="dateRange" stored="true" indexed="true" multiValued="true"/>`
`  <fieldType name="dateRange" class="solr.DateRangeField"/>`
Add sample data and example search settings attached below
[example-search-settings.nt.zip](https://github.com/vivo-project/VIVO/files/11260817/example-search-settings.nt.zip)
Try search, create new document modifiers, search filters

# Interested parties
@VIVO-project/vivo-committers
@chenejac

### CODE REVIEW

**Review by chenejac — APPROVED** [review]
@litvinovg this looks ok to me. Please check my comments in associated Vitro PR.

File: webapp/src/main/webapp/themes/nemo/templates/menu.ftl [hunk 1]
```diff
@@ -37,6 +37,7 @@
 				<input 
 					type="text" 
 					class="form-control" 
+                                        id="filter_input_querytext"
```
Comment (chenejac) [inline webapp/src/main/webapp/themes/nemo/templates/menu.ftl:40]: not sure why specification of this id is needed, didn't find that in some css or somewhere else. Anyway, nemo and tenderfoot might be removed from the VIVO codebase, therefore I wouldn't spend additional time on this.

**Review by chenejac — APPROVED** [review]
@litvinovg looks good to me.

File: webapp/src/main/webapp/themes/nemo/templates/menu.ftl (hunk 1 above)
Comment (litvinovg) [reply webapp/src/main/webapp/themes/nemo/templates/menu.ftl:40]: Id was added to be able to dismiss query text filter value the same any other selected filter could be dismissed.

Comment (chenejac) [reply webapp/src/main/webapp/themes/nemo/templates/menu.ftl:40]: thanks for the clarification

**Review by chenejac — APPROVED** [review]

**Review by ivanmrsulja — APPROVED** [review]
Works as expected. Great work!

**Pull request merged**